### PR TITLE
Move the timer to VR (was in the host before)

### PR DESCRIPTION
**Pull request by irenevanderblij**
By moving the timer to the VR client, we do not have synchronization problems. 

Test this together with the branch timer-to-VR on KroketHost.

### CODE REVIEW

**Comment by Faffie** [comment]
Works fine. I apologise for the spelling mistake. I'm getting old. :(

File: EscapeVR/src/main/java/nl/tudelft/kroket/timer/Timer.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package nl.tudelft.kroket.timer;
+
+import nl.tudelft.kroket.escape.Settings;
+import nl.tudelft.kroket.log.Logger;
+import nl.tudelft.kroket.net.ClientThread;
+import nl.tudelft.kroket.net.protocol.Protocol;
+import nl.tudelft.kroket.screen.HeadUpDisplay;
+
+public class Timer {
```
Comment (alanvanrossum) [inline EscapeVR/src/main/java/nl/tudelft/kroket/timer/Timer.java:14]: Add Javadoc to this class

**Comment by irenevanderblij** [comment]
I added Javadoc comments.
However, the indentation settings are wrong in my eclipse, and I cannot get them right.
Can someone correct the indentation in the Timer class.

**Comment by alanvanrossum** [comment]
sure thing i'll do the indentation

**Comment by alanvanrossum** [comment]
never mind

**Comment by Mayke93** [comment]
this works, indentation is fixed

**Pull request merged**